homeassistant/components/history/__init__.py

```diff
@@ -34,22 +34,21 @@
 import homeassistant.util.dt as dt_util
 
 from . import websocket_api
-from .const import (
-    DOMAIN,
-    HISTORY_ENTITIES_FILTER,
-    HISTORY_FILTERS,
-    HISTORY_USE_INCLUDE_ORDER,
-)
+from .const import DOMAIN
 from .helpers import entities_may_have_state_changes_after
+from .models import HistoryConfig
 
 _LOGGER = logging.getLogger(__name__)
 
 CONF_ORDER = "use_include_order"
 
 CONFIG_SCHEMA = vol.Schema(
     {
-        DOMAIN: INCLUDE_EXCLUDE_BASE_FILTER_SCHEMA.extend(
-            {vol.Optional(CONF_ORDER, default=False): cv.boolean}
+        DOMAIN: vol.All(
+            cv.deprecated(CONF_ORDER),
```

Comment:
Should this raise a repair when detected it is set?

Reply:
It's effectively done nothing for a while now so I'm thinking the cv.deprecated is probably enough but happy to do otherwise if you think it's needed

Reply:
ok, lets continue :)